### DISCUSSION TITLE
brew services show status error

### DISCUSSION THREAD

**Discussion by DRAGON0818** [Homebrew/discussions#917] [Everyday usage]
how to deal it
![image](https://user-images.githubusercontent.com/30966910/108979573-c32be200-76c5-11eb-9610-58c0c6565d68.png)

**Comment by SMillerDev** [comment]
First, please do not post text content as images. It makes it a lot harder to read and indicate issues.

Second, this is a question about basic usage of `brew services` it shouldn't have been marked as "Advanced". I moved it to "General"

Third, this isn't really a brew question at all.  `brew services` just runs the program you have installed. You'll have to check the logs to see if/why it stopped because that is up to each individual program to relay. For mysql the log file is defined in `/etc/my.cnf`, `/etc/mysql/my.cnf` or `/usr/local/etc/my.cnf`.

**Reply by DRAGON0818** [reply]
Thanks very much for your reply.
because  my english is not well.so i post text context as images.
